admin_schema.py

```diff
@@ -0,0 +1,21 @@
+import graphene
```

Comment:
User model not necesary for the sake of decoding a token received from front end as part of response headers